mobile-widgets/qml/TemplateTitle.qml

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.11
+
+
+Item {
+	property alias title: myLabel.text
+	width: parent.width -20
```

Comment:
same comment about using `Kirigami.Units`

Reply:
I corrected this one as well (problem is when we start to use, different layout for e.g. iPad and iPhone), this was my fast way of adding a width.

I have been looking for an example showing width = max width - a bit for padding. If you have an example please point me to it.